README.md

```diff
@@ -130,7 +130,46 @@ Note: during D day. There is a lot of work to be done. The lesser people there a
 
 ### Emcee
 
+#### Expectations
+
+The emcees help transition between sections of the conference. They also engage the audience with additional insight and help facilitate the Q&A. They also provide time for the speaker to get ready. 
+
+#### People
+
+At any point in time, there must be an emcee, and there should be a backup emcee. It is highly recommended to rotate emcee roles each day or each half-day. 
+
+#### Tasks
 
+##### Pre-Conference
+
+- Prepare speaker introduction
+  - Correct pronunciation of speaker names
+- Prepare talk introduction
+- If necessary: Talk and speaker introduction slide to show the audience. Work with the designer.
+
+Emcees should make preparations for themselves. Their own material can come in any form, like slides, notes or cards. [BYOD](https://en.wikipedia.org/wiki/Bring_your_own_device). Verify that there is space to locate your laptop (or in some venues an additional screen).
+
+#### Conference Day 
+
+- Start the day, welcome everyone & other misc briefings
+- Handover to the Chairperson/Vice Chairperson for introductory talk
```

Comment:
Provide ad-hoc updates such as last-minute changes to scheduling or lost and found announcements.